src/test/java/org/broadinstitute/hellbender/utils/svd/SingularValueDecomposerUnitTest.java

```diff
@@ -31,11 +31,7 @@ public void testBasicFunction(final SingularValueDecomposer svder){
         final double[] expectedSingularValues = {3.0, 2.0};
 
         final SVD svd = svder.createSVD(m);
-        Assert.assertEquals(svd.getU().multiply(MatrixUtils.createRealDiagonalMatrix(svd.getSingularValues())).multiply(svd.getV().transpose()), m);
-        Assert.assertTrue(isUnitaryMatrix(svd.getV()), "V is unitary");
-        Assert.assertTrue(isUnitaryMatrix(svd.getU()), "U is unitary");
-        Assert.assertEquals(svd.getV().transpose().multiply(svd.getV()), MatrixUtils.createRealIdentityMatrix(2), "V is unitary");
-        Assert.assertEquals(svd.getU().transpose().multiply(svd.getU()), MatrixUtils.createRealIdentityMatrix(2), "U is unitary");
+        assertSVD(svd, m);
```

Comment:
Not your code, but while you're at it `isUnitaryMatrix` is needlessly complicated.  Unitary matrices, in particular the U and V of an SVD, are by definition square.  Thus the necessary check is simply that the transpose is the inverse i.e. `m.multiply(m.transpose()) == m.transpose().multiply(m) == IdentityMatrix`.